_storage/hub-operation.md

```diff
@@ -0,0 +1,108 @@
+  "proofsConfig": {
+    "proofsRequired" : 0
+  },
+  "pageSize": 20,
+  "bucket": "YOUR_BUCKET_NAME",
+  "awsCredentials": {
+     "accessKeyID": "YOUR_ACCESS_KEY",
+     "secretAccessKey": "YOUR_SECRET_KEY"
+  },
+  "argsTransport": {
+    "level": "debug",
+    "handleExceptions": true,
+    "stringify": true,
+    "timestamp": true,
+    "colorize": false,
+    "json": true
+  }
+}
+```
+
+You can specify the logging level, the number of social proofs required
+for addresses to write to the system, the backend driver, the credentials
+for that backend driver, and the readURL for the storage provider.
+
+### Require the correct hub URL
+
+If you enable on the `requireCorrectHubUrl` option in your `config.json`
+file, your Gaia hub will require that authentication requests
+correctly include the `hubURL` they are trying to connect with. Use this option to prevent a malicious Gaia hub from using an authentication
+token for itself on other Gaia hubs.
+
+By default, the Gaia hub will validate that the supplied URL matches
+`https://${config.servername}`, but if there are multiple valid URLs
+for clients to reach the hub at, you can include a list in your `config.json`:
+
+```javascript
+{
+  ....
+  servername: "normalserver.com"
+  validHubUrls: [ "https://specialserver.com/",
+                  "https://legacyurl.info" ]
+  ....
+}
+```
+
+### The readURL parameter
+
+By default, hub drivers return read URLs that point directly at the written content. For example, an S3 driver would return the URL directly to the S3 file. If you configure a CDN or domain to point at that same bucket, you can use the `readURL` parameter to tell the hub that files can be read from a given URL. For example, the `hub.blockstack.org` Gaia Hub is configured to return a read URL that looks like `https://gaia.blockstack.org/hub/`.
+
+Unset the `readURL` parameter if you do not intend to deploy any caching.
+
+### The proofsRequired parameter
+
+You can configure a hub to require a minimum number of social proofs in a user's profile to accept writes from that user. You can use this setting as a crude spam-control mechanism. However, for the smoothest operation of your Gaia hub, set the `proofsConfig.proofsRequired` value to `0`.
+
+
+## Open or private hubs
+
+You can configure an open-membership storage hub or a private storage hub. An open-membership hub, as it sounds, allows any user to use the hub service. A private hub limits the use of the service. In this section, you learn about configuring each type.
+
+### Open-membership hub
+
+An open-membership storage hub permits writes for _any_ address top-level
+directory. Every request is validated such that write requests must provide
+valid authentication tokens for that address. Operating in this mode is
+recommended for service and identity providers who wish to support many
+different users.
+
+To limit the users that may interact with such a hub, you can limit users to
```

Comment:
Similarly -- I'd prefer if we didn't talk about social proofs, because I'm not sure that this feature works at the moment.

Reply:
Sure. Removed mention from all Hub docs.